src/main/resources/assets/reignofnether/lang/en_us.json

```diff
@@ -49,7 +49,7 @@
     "abilities.reignofnether.connect_portal.tooltip1": "Connect to a transport portal you own anywhere else.",
     "abilities.reignofnether.connect_portal.tooltip2": "Overrides any existing connections on both sides.",
     "abilities.reignofnether.connect_porta.error1": "Must target another completed transport portal you own.",
-    "abilities.reignofnether.disconnect_portal": "Sever Connection",
+    "abilities.reignofnether.disconnect_portal": "Server Connection",
```

Comment:
not a typo, its disconnection,

Reply:
Yes, this is for the tooltip for a transport portal to cut off its connection to another portal (ie. severing it)